docs/_documentations/troubleshooting.md

```diff
@@ -206,7 +206,7 @@ Issue link: https://github.com/eclipse/codewind/issues/243
 ## Adding an existing Open Liberty project fails to build because of missing files
 An Open Liberty project fails to build after it is added into Codewind with the **Add Existing Project** action, and the project fails to build because of missing files.
 
-**Workaround:** Bind the existing project again and click **No** followed by **Other** for the project type.
+**Workaround:** Bind the existing project again but this time, don't accept the detected type. Click **No** for VSCode, **Next** for Eclipse, and then choose **Other (Codewind Basic Container)** for the project type.
```

Comment:
IntelliJ should have a flow like this too, but I'm not sure what the button would say. We could simplify this by not specifying exactly which buttons have to be pressed in each IDE.

Reply:
Reworded